<?xml version="1.0"?>

<launch>
    <node pkg="face_spoofing" type="">

</launch>
